index.js

```diff
@@ -1 +1,94 @@
-console.log('Hello World!');
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const chai = require('chai');
+
+const should = chai.should();
+
+/**
+ * Validates the structure of the "proof" property on a digital document.
+ *
+ * @param {object} options - Options to use.
+ * @param {object} options.data - A digital document with "proof" property on
+ *   it.
+ * @param {string} [options.vendorName] - The name of the vendor who issued the
+ *   document.
+ *
+ * @returns {undefined} Just returns on success.
+ */
+function checkDataIntegrityProofFormat({data, vendorName} = {}) {
+  if(!data) {
+    throw new Error('"data" object is required.');
+  }
+  if(vendorName && typeof vendorName !== 'string') {
```

Comment:
It seems like this should be required?

```suggestion
  if(!(vendorName && typeof vendorName === 'string')) {
```

Reply:
Updated [7cfc3ba](https://github.com/digitalbazaar/data-integrity-test-suite-assertion/pull/1/commits/7cfc3ba0fd2e6f9b86c59980d019f6772c9778b8)